<launch>
		
  	<arg name="vehicle_name" default="oshkosh1"/>

        <node name="$(arg vehicle_name)_vel_controller"  type="oshkosh_vel_controller.py"  pkg="oshkosh" args="$(arg vehicle_name)" output="screen" respawn="true"/>

        <node name="$(arg vehicle_name)_wp_controller"   type="oshkosh_wp_controller.py"   pkg="oshkosh" args="$(arg vehicle_name)" output="screen" respawn="true"/>

        <node name="$(arg vehicle_name)_path_controller" type="oshkosh_path_controller.py" pkg="oshkosh" args="$(arg vehicle_name)" output="screen" respawn="true"/>



</launch>        
        
